File: 1051-height-checker/1051-height-checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int count =0;
        int[] arr = heights.clone(); 
        Arrays.sort(arr); 
        
        for(int i = 0; i<= heights.length-1; i++){
            if(heights[i]!= arr[i]){
                count +=1; 
            }
        }
        return(count);
    }
}

File: 1051-height-checker/1051-height-checker.py
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        arr = sorted(heights)
        #print(arr)
        count = 0 
        for i in range(len(heights)):
            if(arr[i]!= heights[i]):
                count+=1 
        
        return count